utils/post_utils.jsx

```diff
@@ -484,22 +484,6 @@ export function getNewMessageIndex(postListIds) {
     );
 }
 
-export function makeGetReplyCount() {
```

Comment:
I very much appreciate removing this since selectors being defined in `utils` kind of annoys me since they have their own folder already 😛